front/app/components/IdeaForm/index.tsx

```diff
@@ -749,6 +754,26 @@ class IdeaForm extends PureComponent<
         phases
       );
 
+      const AdminBudgetFieldLabel = () => {
```

Comment:
Not quite sure if this is the correct way to handle this? Having this code in the FormLabel directly seemed messy though.

Reply:
I think it is fine for now

Theoretically ,something like this might work 🤔 instead of a component.
```
      const adminBudgetFieldLabel = <>
            <FormattedMessage
              {...messages.budgetLabel}
              values={{
                currency: tenantCurrency,
                maxBudget: pbContext?.attributes.max_budget,
              }}
            />
            <IconTooltip
              iconColor="black"
              marginLeft="4px"
              icon="admin"
              content={<FormattedMessage {...messages.adminFieldTooltip} />}
            />
          </>
```